server/lib/accounts.js

```diff
@@ -95,6 +95,13 @@ Accounts.insertUserDoc = _.wrap(Accounts.insertUserDoc, function(insertUserDoc,
 
 	delete user.globalRoles;
 
+	if (!(user.services && user.services.password)) {
```

Comment:
This is a little bit confused to me, why not:
`if (!user.services || !user.services.password) {`